core-bundle/contao/library/Contao/TemplateTrait.php

```diff
@@ -191,6 +191,26 @@ public function addCspHash(string $directive, string $script, string $algorithm
 		$csp->addHash($directive, $script, $algorithm);
 	}
 
+	/**
+	 * Adds a CSP hash for a given inline style and also adds the 'unsafe-hashes' source to the directive automatically.
+	 */
+	public function cspInlineStyle(string $style, string $algorithm = 'sha384'): string
```

Comment:
I assume this also handles multiple styles such as `text-decoration: underline; font-weight: bold`, doesn‘t it?

Reply:
The hash needs to be computed for the whole content of the `style` attribute, yes.